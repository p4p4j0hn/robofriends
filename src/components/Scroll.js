import React from 'react';

const Scroll = (props) => {
  return(
    <div style={{ overflowY: 'scroll', flex: '0 1 auto' }}>
      {props.children}
    </div>
  );
}

export default Scroll;
