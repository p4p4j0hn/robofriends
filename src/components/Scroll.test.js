import React from 'react';
import { shallow } from 'enzyme';
import Scroll from './Scroll';

describe('scroll', () => {
  it('expect to render Scroll component', () => {
    expect.assertions(1);
    expect(shallow(<Scroll />)).toMatchSnapshot();
  });
});
