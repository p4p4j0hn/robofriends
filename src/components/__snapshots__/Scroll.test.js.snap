// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`scroll expect to render Scroll component 1`] = `
<div
  className="hideScrollBar"
  style={
    Object {
      "flex": "0 1 auto",
      "overflowY": "scroll",
    }
  }
/>
`;
